### ISSUE TITLE
OpenFL arguments not persisted in module settings

### ISSUE THREAD

**Issue by jasonsturges** [TiVo/intellij-haxe#74]
When I enter _OpenFL arguments_ for a module, the _apply_ button does not enable nor do changes persist.  

Even if I change another parameter to enable the _apply_, closing the _Project Structure_ dialog by pressing _OK_ and immediately opening it again, the _OpenFL arguments_ input is empty.

Below, I've entered _-debug_ in the _OpenFL arguments_ input field:

![image](https://cloud.githubusercontent.com/assets/1213591/5372771/1c36f38a-8014-11e4-9460-f3f3f66b4148.png)

This occurs with both IntelliJ 13 and IntelliJ 14, with either the intellij-haxe 0.8.1.1.TiVo.3 release and 0.8.1 hosted from JetBrains IntelliJ plugin repository.

TIR: STB-7304

**Comment by hopewise** [comment]
I think the developers can fix this issue easily, I hope they fix it very soon ..

**Comment by as3boyan** [comment]
Yeah, that should be easy to fix, I need to check it in latest ClassHierarchy branch.

**Comment by profelis** [comment]
any progress?

**Comment by EBatTiVo** [comment]
There has been no progress on this issue.  Other work has taken priority.  I understand this is truly a nuisance for those of you who use the dialog to set up your projects.  The workaround is to edit the .iml files outside of the IDE.  Here is an .iml that builds OpenFL for your reference:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<module type="HAXE_MODULE" version="4">
  <component name="HaxeModuleSettingsStorage">
    <option name="arguments" value="-D debug -debug -D HXCPP_DEBUG
    <option name="buildConfig" value="3" />
    <option name="excludeFromCompilation" value="true" />
    <option name="openFLFlags" value="-v -debug -Ddisable_watchdog -D HXCPP_DEBUG
    <option name="openFLPath" value="$MODULE_DIR$/../Export/core/tasks/standby/lib/project.xml" />
    <option name="openFLTarget" value="Linux" />
  </component>
  <component name="NewModuleRootManager" inherit-compiler-output="false">
    <output url="file://$MODULE_DIR$/bin" />
    <output-test url="file://$MODULE_DIR$/bin" />
    <exclude-output />
    <content url="file://$MODULE_DIR$/../../../srcroot/core/tasks/standby/lib">
      <sourceFolder url="file://$MODULE_DIR$/../../../srcroot/core/tasks/standby/lib" isTestSource
    </content>
    <content url="file://$MODULE_DIR$/../../../srcroot/core/tasks/standby/lib/.tmp">
      <excludeFolder url="file://$MODULE_DIR$/../../../srcroot/core/tasks/standby/lib/.tmp" />
    </content>
    <orderEntry type="sourceFolder" forTests="false" />
    <orderEntry type="inheritedJdk" />
    <orderEntry type="module" module-name="core_shared_interfaces" exported="" />
    <orderEntry type="module" module-name="core_shared_task" exported="" />
    <orderEntry type="module" module-name="core_tasks_suspending" exported="" />
    <orderEntry type="library" scope="PROVIDED" name="openfl-lib-2,0,0" level="project" />
  </component>
</module>
```

**Comment by profelis** [comment]
@EBatTiVo Thanks! `<option name="openFLFlags"` works perfect. Now I can use `--connect ` with openfl projects

**Comment by EBatTiVo** [comment]
Fixed: fe5de67f621beae9f08d8032bf38571224c6de4b  PR #401 

Will be in 0.9.9 release.
